<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-6U5DIVL" failures="2" tests="5" name="Test" time="69.718" errors="0" timestamp="2024-02-25T00:47:24 IST">
  <testcase classname="testCases.TC_003_LoginDDT" name="verify_loginDDT" time="12.028">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC_003_LoginDDT.verify_loginDDT(TC_003_LoginDDT.java:45)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="testCases.TC_003_LoginDDT" name="verify_loginDDT" time="13.21"/>
  <testcase classname="testCases.TC_003_LoginDDT" name="verify_loginDDT" time="11.482"/>
  <testcase classname="testCases.TC_003_LoginDDT" name="verify_loginDDT" time="13.27"/>
  <testcase classname="testCases.TC_003_LoginDDT" name="verify_loginDDT" time="11.296">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC_003_LoginDDT.verify_loginDDT(TC_003_LoginDDT.java:45)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
</testsuite> <!-- Test -->
